client/src/components/DirectoryAddUpdate.jsx

```diff
@@ -747,17 +758,32 @@ export default function DirectoryAddUpdate() {
                   ]}
                 />
               </div>
-              <div className="form-group col-md-6" align="center">
+
+              {coopRejectComplete ?
+              <>
+                <h2>The coop rejection has been submitted.</h2>
+                <Link to={"/unapproved-list/"}>Return to unapproved coops list</Link>
+              </>
+              :
+              <>
+                <div className="form-group col-md-6" align="center">
+                  {approvalForm ? 
+                      <Button buttonType={"primary"} title={"Approve"} type={"submit"} />
+                    :
+                      <Button buttonType={"primary"} title={"Send Addition/Update"} type={"submit"} />
+                  }
+                </div>
+                <div className="form-group col-md-6" align="center">
                 {approvalForm ? 
-                    <Button buttonType={"primary"} title={"Approve"} type={"submit"} />
+                    <RejectForm id={id} setCoopRejectComplete={setCoopRejectComplete} />
```

Comment:
This might be just a placeholder, but wouldn't we want to put a button that says "Reject" here that spawns the modal with the form?  It looks like we're includign the form component itself, but I may be misreading waht's going on.